src/main/lombok/eu/markov/jenkins/plugin/mvnmeta/MavenMetadataParameterDefinition.java

```diff
@@ -374,6 +430,14 @@ private SortOrder(String displayName) {
   public static class DescriptorImpl extends ParameterDescriptor {
 
     public FormValidation doCheckVersionFilter(@QueryParameter String value) {
+      return this.doCheckRegex(value);
```

Comment:
Personal coding style. I like it if I see instantly if a static or instance member is invoked.

BTW: Just noticed that the method ``doCheckRegex`` could be static.

I could do either change if desired. It's not really important to me ...